### PR TITLE
Single scroll demo

### PR DESCRIPTION
**Pull request by moniika**
Adding a demo for single-direction scrollbars.

### CODE REVIEW

File: examples/single-direction-scroll/index.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Blockly Demo: Single-direction Scrolling</title>
+  <!--  TODO: Change back to unpkg and remove relative path before merging -->
```
Comment (samelhusseini) [inline examples/single-direction-scroll/index.html:6]: Looks like this TODO should be done now?

File: examples/single-direction-scroll/package.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "single-direction-scroll",
```
Comment (samelhusseini) [inline examples/single-direction-scroll/package.json:2]: add -demo to the end of the name for consistency with other demos.

**Review by samelhusseini — APPROVED** [review]
Looks good (once comments are resolved)

File: examples/single-direction-scroll/index.html (hunk 1 above)
Comment (moniika) [reply examples/single-direction-scroll/index.html:6]: done

File: examples/single-direction-scroll/package.json (hunk 2 above)
Comment (moniika) [reply examples/single-direction-scroll/package.json:2]: done

**Pull request merged**